deployments/nginx-behind-envoy-deployment.yaml

```diff
@@ -106,22 +116,18 @@ spec:
             capabilities:
               add: ["IPC_LOCK"]
           command:
-            - "/envoy-static"
+            - "/usr/local/bin/envoy-static"
           args:
             - "-c"
             - "/etc/envoy/config/envoy-conf.yaml"
             - "--cpuset-threads"
-          env:
-            - name: "QAT_SECTION_NAME"
-              value: "SHIM"
-            - name: "POLL_DELAY"
-              value: "2000000"
           resources:
             limits:
-              qat.intel.com/generic: 1
+              qat.intel.com/cy2_dc2: 1
```

Comment:
my preference is to keep `generic`. for the intel-qat-kernel-plugin resources we could create a kustomization overlay that patches the resource type.